packages/manager/modules/vps/src/migration/vps-migration.constants.js

```diff
@@ -0,0 +1,49 @@
+export const EXCLUDED_VERSIONS = ['2014v1', '2013v1'];
+export const MIGRATION_STATUS = {
+  NOT_AVAILABLE: 'notAvailable',
+  PLANNED: 'planned',
+  TO_PLAN: 'toPlan',
+};
+export const EXPIRY_DATE = '27-07-2020';
```

Comment:
Same constant than https://github.com/ovh/manager/pull/2911/files#diff-55e69ab7bbcfc08daf2fcdee001c0f86R11 ?

Reply:
It was. But it has now been changed :)